code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -76,7 +77,7 @@
 		"Skrell" = 'icons/mob/species/skrell/suit.dmi',
 		"Vox" = 'icons/mob/species/vox/suit.dmi',
 		"Vulpkanin" = 'icons/mob/species/vulpkanin/suit.dmi',
-		"Drask" = 'icons/mob/species/drask/suit.dmi'
+		"Drask" = 'icons/mob/species/drask/suit.dmi',

```

Comment:
huh... it can be removed, sure. i don't remember how that ended up there. how should i go about fix these, by the way? edit the files locally and recommit?